iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/converter/ProjectToOrganizationConverter.java

```diff
@@ -0,0 +1,70 @@
+package eu.dnetlib.iis.wf.importer.converter;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NavigableMap;
+
+import org.apache.hadoop.hbase.client.Result;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+
+import eu.dnetlib.data.proto.OafProtos.Oaf;
+import eu.dnetlib.data.proto.OafProtos.OafRel;
+import eu.dnetlib.iis.importer.schemas.ProjectToOrganization;
+import eu.dnetlib.iis.wf.importer.OafHelper;
+import eu.dnetlib.iis.wf.importer.input.approver.ResultApprover;
+
+/**
+ * {@link ProjectToOrganization} converter.
+ * @author mhorst
+ *
+ */
+public class ProjectToOrganizationConverter extends AbstractAvroConverter<ProjectToOrganization[]> {
+
+	/**
+	 * Project-organization relation column family.
+	 */
+	private final byte[] relationColumnFamilyBytes;
+	
+	/**
+	 * Default constructor.
+	 * @param encoding encoding to be used when converting byte[] to String
+	 * @param resultApprover relation approver
+	 * @param relationColumnFamilyBytes project-organization column family
+	 */
+	public ProjectToOrganizationConverter(String encoding,
+			ResultApprover resultApprover,
+			byte[] relationColumnFamilyBytes) {
+		super(encoding, resultApprover);
+		this.relationColumnFamilyBytes = OafHelper.copyArrayWhenNotNull(
+				relationColumnFamilyBytes);
+	}
+
+	@Override
```

Comment:
Added, although it was neither my nor my eclipse habit to comment overridden methods. Anyway I've just read at some sites it is advised to put comments in such scenario. Strangely eclipse refuses to assist with contextual help and auto linking in this context.